src/groups/mwc/mwcstm/mwcstm_values.h

```diff
@@ -116,109 +115,88 @@ class StatContextConfigurationChoice {
     static const bdlat_SelectionInfo SELECTION_INFO_ARRAY[];
 
     // CLASS METHODS
-
-    /// Return the most current `bdex` streaming version number supported by
-    /// this class.  See the `bslx` package-level documentation for more
-    /// information on `bdex` streaming of value-semantic types and
-    /// containers.
-    static int maxSupportedBdexVersion();
-
-    /// Return selection information for the selection indicated by the
-    /// specified `id` if the selection exists, and 0 otherwise.
     static const bdlat_SelectionInfo* lookupSelectionInfo(int id);
+    // Return selection information for the selection indicated by the
```

Comment:
The file was not regenerated for a long time, and it seems that the generator script has changed much since then. I don't see any option for docs style in `bas_codegen.pl`

Reply:
Okay.  I'll make a separate PR to run the Doxygen conversion script on them.  I'm not sure if it makes sense for `bas_codegen.pl` to support Doxygen style either, but we can at a later date figure out if it makes sense for us to always run the Doxygen conversion script after `bas_codegen.pl`.

This is in MWC, which we don't generate documentation for, so at the moment this is fine, if inconsistent.